_hardware/controllers.md

```diff
@@ -214,10 +214,31 @@ groups:
       description:
 
     - name: MKS_LCD12864A
-      description:
+      description: |
+        Designed for 5v 8bit mootherboards.*
+        - 128x64 pixel LCD panel
+        - SD Card reader
+        - Reset button
+        - Combined rotary encoder and push button for menu control
+        - Speaker
```

Comment:
```suggestion
        - Buzzer
```
I don't want to keep arguing here, but I still think both MKS LCD12864A and B have a buzzer, not speaker. They *do* work with `#define SPEAKER`, but they also *do* make beeping sounds without that, which a 'true' speaker just can't do.

Reply:
Well, yes, they *can* work without enabling `SPEAKER`, but they don't *require* users to enable it (while some other controllers do, like Fysetc Mini12864). IMO, marking these as speakers would just lead to confusion.

Reply:
We don't have a sanity check that requires them to be defined as such. Having "Speaker" mentioned clues the user into what features are supported.

Speaker is correct here because they can play tones and not just single/monotone beeps.